src/components/ProfileGamesTable.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import StarIcon from "@material-ui/icons/Star";
+import amber from "@material-ui/core/colors/amber";
+import grey from "@material-ui/core/colors/grey";
+import moment from "moment";
+
+import ElapsedTime from "./ElapsedTime";
+import User from "./User";
+import Loading from "./Loading";
+
+const useStyles = makeStyles((theme) => ({
+  gamesTable: {
+    display: "flex",
+    flexDirection: "column",
+    maxHeight: 400,
+    marginBottom: theme.spacing(1),
+    whiteSpace: "nowrap",
+    "& td, & th": {
+      paddingTop: 6,
+      paddingBottom: 6,
+      paddingLeft: 12,
+      paddingRight: 12,
+    },
+    "& svg": {
+      display: "block",
+    },
+    "& tr:hover": {
+      background: theme.palette.action.hover,
+      cursor: "pointer",
+    },
+  },
+  // Remove cells of some columns of table for small screens
+  vanishingTableCell: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+  },
+}));
+function ProfileGamesTable({ userId, gamesData, handleClickGame }) {
+  const classes = useStyles();
+  if (!gamesData) {
+    return <Loading />;
+  }
+  if (Object.keys(gamesData).length === 0) {
+    return (
+      <Typography style={{ color: grey[300] }}>
+        No games finished yet...
+      </Typography>
+    );
+  }
+  return (
+    <TableContainer component={Paper} className={classes.gamesTable}>
+      <Table size="small" stickyHeader>
+        <TableHead>
+          <TableRow>
+            <TableCell>#</TableCell>
+            <TableCell className={classes.vanishingTableCell}>Host</TableCell>
+            <TableCell>Players</TableCell>
+            <TableCell>Num. Sets</TableCell>
+            <TableCell>Length</TableCell>
+            <TableCell className={classes.vanishingTableCell}>
+              Created
+            </TableCell>
+            <TableCell>Won</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {Object.values(gamesData)
+            .sort((g1, g2) => g2.createdAt - g1.createdAt)
+            .map((game, i) => {
+              return (
+                <TableRow key={i} onClick={() => handleClickGame(game.gameId)}>
```

Comment:
If you `import { Link as RouterLink } from "react-router-dom";` at the top, then you could do away with the click handler and redirects entirely. Instead, 
```suggestion
                <TableRow key={i} component={RouterLink} to={`/game/${game.gameId}`}>
```

Reply:
I'm not sure if this works for TableRow, when I tried it I got  "<td> cannot appear as a child of <a>".